### PR TITLE
[17.0][FIX] impersonate_login: fix user_id comodel_name

### PR DESCRIPTION
**Pull request by astirpe**

### CODE REVIEW

**Review by Kev-Roche — APPROVED** [review]

**Review by CasVissers-360ERP — APPROVED** [review]

**Review by rvalyi — APPROVED** [review]

File: impersonate_login/models/impersonate_log.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ class ImpersonateLog(models.Model):
     _description = "Impersonate Logs"
 
     user_id = fields.Many2one(
-        comodel_name="res.partner",
+        comodel_name="res.users",
```
Comment (simahawk) [inline impersonate_login/models/impersonate_log.py:14]: does it require any migration for existing records?

Comment (astirpe) [reply impersonate_login/models/impersonate_log.py:14]: @simahawk this is the fix already merged in V16 https://github.com/OCA/server-auth/commit/51eccf805efebcdd1f3c0141619a76a88991bcea

The integer written in `user_id` is a `res.users` anyway, so no need of migration

**Pull request merged**